athena/templates/create_table_tdr_cloudtrail_logs_mgmt.sql.tpl

```diff
@@ -0,0 +1,49 @@
+CREATE EXTERNAL TABLE tdr_cloudtrail_logs_mgmt (
+  eventversion STRING,
+  useridentity STRUCT<
```

Comment:
If it's allowed by the template language, would you mind adding some indentation to show where each `STRUCT` starts and ends?

Reply:
thanks, updated